src/index.html

```diff
@@ -7,11 +7,176 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <title>HTML Form</title>
+
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div class="labl">
+          <label for="surname">Surname:</label>
+          <input type="text"
+                 id="surname"
+                 name="surname"
+                 required
+          >
+        </div>
+
+        <div class="labl">
+          <label for="name">Name:</label>
+          <input type="text"
+                 id="name"
+                 name="name"
+                 required
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            id="name"
            name="name"
            required
          >
```
Форматируй теги таким образом